### PR TITLE
Common readme

### PR DESCRIPTION
**Pull request by jwesterd-redhat**

### CODE REVIEW

**Comment by jwesterd-redhat** [comment]
Updated to localize the scope and limit unneeded variables.

File: roles/common/README.md [hunk 1]
```diff
@@ -1,21 +1,45 @@
...
-|-------------------------|-----------------------|-----------------------------------------------------|
-| job_name                |                       | String to describe the job run.                     |
-| log_directory           | "/var/log/ripu"       | Directory under which local log files will be written. This directory will be created is it is not already present. |
-| log_file                | "{{ log_directory }}/ripu.log" | Local log filename. When a playbook job finishes, a timestamp suffix is appended to the end of the filename. |
+# Requirements
 
-## Example playbook
+Define job_name in the import as seen below
 
-This is a common role included by the `analysis` and `upgrade` roles. It does not need to be explicitly included in your playbook.
```
Comment (swapdisk) [inline roles/common/README.md:17]: "This is a common role included by the `analysis` and `upgrade` roles. It does not need to be explicitly included in your playbook." still needs to be said.

File: roles/common/README.md [hunk 2]
```diff
@@ -1,21 +1,45 @@
...
-## License
+ # Logging
+
+Logs will accumulate in the directory referenced by logfile, with a suffixed datestamp upon completion.
+
+If a log file exists during execution of this role (without suffixed datestamp), execution will terminate as there is an analysis job running already.
+
+Logs will not survive a rollback. They need to be removed off the system prior to a snapshot revert.
+
+# Example invocation
```
Comment (swapdisk) [inline roles/common/README.md:28]: Remove the example invocation.

File: roles/common/README.md [hunk 3]
```diff
@@ -1,21 +1,45 @@
...
+# Requirements
 
-## Example playbook
+Define job_name in the import as seen below
 
-This is a common role included by the `analysis` and `upgrade` roles. It does not need to be explicitly included in your playbook.
+# Role variables
+| Name                  | Type | Optional? | Default value | Description                                         |
+|-----------------------|--------|-----------|---------------|-------------------------------------------------|
+| job_name | String | None | The string describes the job run |
```
Comment (swapdisk) [inline roles/common/README.md:16]: Columns messed up. Fix like this... 

```suggestion
| job_name | String | No | None | The string describes the job run |
```

**Review by swapdisk — CHANGES_REQUESTED** [review]
Please make suggested changes and then will be GTM.

File: roles/common/README.md (hunk 1 above)
Comment (jwesterd-redhat) [reply roles/common/README.md:17]: Ok

File: roles/common/README.md (hunk 2 above)
Comment (jwesterd-redhat) [reply roles/common/README.md:28]: OK

File: roles/common/README.md (hunk 3 above)
Comment (jwesterd-redhat) [reply roles/common/README.md:16]: OK

**Review by jwesterd-redhat — COMMENTED** [review]
fixed last line. took periods out of the tables

**Review by swapdisk — APPROVED** [review]

**Pull request merged**